File: Modul 7/Soal 5/PRAK605-2210817220007-AISYA ANINDIA SEPTHA.py
a = int (input())
b = []
print('Matriks A')
for c in range (a) :
    d = list (map(int, input().split()))
    b.append(d)
e = []
print('Matriks B')
for c in range (a) :
    d = list (map(int, input().split()))
    e.append(d)
print()
f = []
print('Matriks AXB')
for c in range (a) :
    f.append([])
    for g in range (a) :
        h = 0
        for d in range (a) :
            h = h + b[c][d] * e[d][g]
        f[c].append(h)
for i in f :
    for d in i :
        print(d, end=' ')
    print()
print()

File: Modul 7/Soal 5/PRAK605-2210817220007-AISYA ANINDIA SEPTHA.c
#include <stdio.h>
int main()
{
int a[10][10], b[10][10], c[10][10], d, e, f, g, h;
h = 0;
scanf("%d", &g);
printf("Matriks A\n");
for(d=0; d<g; d++)
    {for(e=0; e<g; e++)
        {scanf("%d", &a[d][e]);}
    }
printf("Matriks B\n");
for(d=0; d<g; d++)
    {for(e=0; e<g; e++)
        {scanf("%d", &b[d][e]);}
    }
for(d=0; d<g; d++)
    {for(e=0; e<g; e++)
        {for(f=0; f<g; f++)
            {h = h + a[d][f] * b[f][e];}
        c[d][e] = h;
        h = 0;}
    }
printf("\nMatriks AXB\n");
for(d=0; d<g; d++)
    {for(e=0; e<g; e++)
        {printf("%d ", c[d][e]);}
    printf("\n");
    }
return 0;
}
